modules/virt-about-eus-updates.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-eus-updates_{context}"]
+= About EUS-to-EUS updates
+
+Every even-numbered minor version of {product-title}, including 4.10 and 4.12, is an Extended Update Support (EUS) version. However, because Kubernetes design mandates serial minor version updates, you cannot directly update from one EUS version to the next. You must first update to the each z-stream release of the odd-numbered minor version before you can proceed to the target EUS version.
```

Comment:
`You must first update to the each z-stream release of the odd-numbered minor version before you can proceed to the target EUS version.` Should this be elaborated a bit more? e.g. After upgrading to the non-eus z-stream, from source eus version, one must upgrade through all the available z-streams of this minor version, before one can upgrade to target EUS version.